### ISSUE TITLE
[Feature Request] Able to specify avs file for compaction

### ISSUE THREAD

**Issue by windkit** [leo-project/leofs#456]
### Description
LeoFS admins may want to specify compaction target on avs basis (e.g. `1.avs on Storage Node 1`)

The rationale is as follow:
1. Administrators can spread the compaction tasks across nodes, time of day in a finer manner (each operation is shorted), therefore reduce the impact on user
2. Some AVS files may have much less active objects (when some files are frequently rewritten)

**Comment by PatelParas** [comment]
:+1:

**Comment by yosukehara** [comment]
I've filed [a new issue](https://github.com/leo-project/leofs/issues/460) instead of this because users need to recognize specifing avs file for compaction with ``du detail`` command before executing this. So it's complicated, I believe.

**Comment by windkit** [comment]
It is good to improve the auto-compaction, but to me, it is still better to leave the possibility to users.

One of the biggest reasons to propose this feature is to allow users breakdown compaction into smaller units so they can distribute the task based on their knowledge of the workload pattern.

And I think it is possible to share much of the code base for the proposed improvement and this request. Please consider it.

**Comment by yosukehara** [comment]
@windkit It is able to realize your request with executing only this command after we implemented leo-project/leofs/issues/460

```bash
$ leofs-adm data-compact start <NODE> 1
```

* reference: [compact-start](http://leo-project.net/leofs/docs/admin_guide/admin_guide_5.html#compact-start-storage-node-all-num-of-targets-num-of-compaction-procs)

**Comment by windkit** [comment]
@yosukehara If I don't misunderstand the command, it means "execute compaction on the avs with the lowest active ratio".

Still, i can't understand why not also give the choice to users.
Selecting the one would reclaim most for GC is one of good heuristics but it doesn't mean it fits all needs. 
It is easy to think about scenario like "I want to replace HDD 1 for 1.avs, so I would like to perform compaction first to reduce the amount of data need to be copied"

No matter what, I would revise this after #460

**Comment by yosukehara** [comment]
@windkit #460 means as below example:

* Example:
    * Ratio of active size: 
        * 0.avs: 50.0%
        * 1.avs: 25.0%
        * 2.avs: 35.0%
        * 3.avs: 45.0%
    * Compaction Order:
        * 1 -> 2 -> 3 -> 0

```
$ leofs-adm data-compact start <NODE> 1
## Compact "1.avs"
$ leofs-adm data-compact start <NODE> 3
## Compact "2.avs", "3.avs" and "0.avs"
```

Again, AVS-files reaches autonomic_op.compaction.threshold_active_size_ratio, LeoFS starts the auto-compaction for its AVS-file so that LeoFS can reduce a disk usage more efficiently.

**Comment by yosukehara** [comment]
And also, Actually, it's necessary to support the function we delivered once. Regarding ROI(Return On Investment) of this feature, it's low and only #460 is enough to users, I think.
